Bump airOS to 0.2.6 improving device class matching more devices

CoMPaTech opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->

## Proposed change
Bump airOS module to 0.2.6 - Documentation PR not added. I suggest we wait feedback on the issue before adding their devices as 'supported devices'. Rest assured (and as can be seen from the changelog) both the LiteAP and Nanobeam user data + fixtures are confirmed working in the updated module.

- 0.2.6 introduces
  - support for reported LiteAP GPS and NanoBeam as per issue linked
  - redacting of JSON in exceptions (as in the same data we redact in diagnostics already)
  - fixture is still the device (only renamed and adjusted for 0.2.5 functionality and key-sorted)
- 0.2.5 contains mainly discovery (as in config_flow) preparation and rework of sensors to binary_sensors (as per previous PR) so unrelated to this PR and not affecting

https://github.com/CoMPaTech/python-airos/compare/v0.2.4...v0.2.6 / [Markdown Changelog](https://github.com/CoMPaTech/python-airos/blob/main/CHANGELOG.md)

## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [x] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Deprecation (breaking change to happen in the future)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #150118 #150170 #150134 and https://github.com/home-assistant/home-assistant.io/issues/40343
- This PR is related to issue: 
- Link to documentation pull request: 
- Link to developer documentation pull request: 
- Link to frontend pull request: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] I have followed the [perfect PR recommendations][perfect-pr]
- [x] The code has been formatted using Ruff (`ruff format homeassistant tests`)
- [x] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [x] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [x] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [x] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone+-status%3Afailure

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/development_checklist/
[manifest-docs]: https://developers.home-assistant.io/docs/creating_integration_manifest/
[quality-scale]: https://developers.home-assistant.io/docs/integration_quality_scale_index/
[docs-repository]: https://github.com/home-assistant/home-assistant.io
[perfect-pr]: https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr

CoMPaTech committed:
Bump airOS to 0.2.6 improving device class matching more devices

CoMPaTech committed:
Bump CI for PyPi

CoMPaTech committed:
Bump CI for roborock issue reported

edenhaus commented:
Please explain what 
> Documentation not updated as user should confirm working
means?
Should we wait until users give feedback?

CoMPaTech commented:
Reworded the summary to reflect documentation approach

joostlek commented:
But did you test these changes on the models which are added now?

CoMPaTech commented:
In more detail:
- [x] #150118  - user submitted actual user data - fixture tested in 0.2.6
  - [x] `sta`tions (childs) can have actually have `ptmp` mode
  - [x] `ip6addr` should be an optional key
  - [x] some `remote`s/childs can have no `mode` set (or detected)
  - [x] unknown `flex_mode` added to the dataclass
  - [x] added optional GPS values
- [x] #150170 - ip6addr fixed from the first issue listed
- [x] docs 40343 - relates to sta-ptmp not being an Enum and mode not present in (some) remotes (fixed from the first issue)
- [x] added data-redaction of data in the exception logging (such as GPS positions)

edenhaus reviewed (approved):
Thanks @CoMPaTech 👍

fireheadman commented:
when do we get to test this?
